scripts/rss/build-xml.js

```diff
@@ -0,0 +1,36 @@
+import builder from 'xmlbuilder'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'xmlbuilder'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->